### DISCUSSION TITLE
What is "scale v prediction loss" and how do i use it?

### DISCUSSION THREAD

**Discussion by TeKett** [kohya-ss/sd-scripts#915] [Q&A]
It errors saying it can only be used by enabling "v_parameterization", the only info i find about this is that another project kohya-trainer "added --v_parameterization" but thats it. And searching with literal only brings me to a sd-scripts search page with updated info that no longer has the line of text google is referencing.

**Comment by miabrahams** [comment]
This is for training models using V-parameterization. These models should be labeled "vpred" or "vparam". Most models use standard parameterization so you likely want to leave it off.


https://medium.com/@zljdanceholic/three-stable-diffusion-training-losses-x0-epsilon-and-v-prediction-126de920eb73
